encoding/da.go

```diff
@@ -269,3 +269,54 @@ func TxsToTxsData(txs types.Transactions) []*types.TransactionData {
 	}
 	return txsData
 }
+
+// Fast testing if the compressed data is compatible with our circuit
+// (require specified frame header and each block is compressed)
+func TestCompressedDataCompatibility(data []byte) error {
```

Comment:
I don't fully understand how we are supposed to use `TestCompressedDataCompatibility`.

You mentioned ccc, but we don't run ccc inside `rollup-relayer` (implementation detail).

But it sounds like you suggest calling this function on `compressedBlobBytes` (that we produce [here](https://github.com/scroll-tech/da-codec/blob/5b6cded48ab789c6c7210f113bbea900c32734e2/encoding/codecv2/codecv2.go#L181)), is that correct?

Reply:
Yes. Or more specificied, in `EstimateBatchL1CommitBatchSizeAndBlobSize`?

If the test had failed, it can be handled just like the blobBytes has exceed the limit